src/makeDefaultEffects.js

```diff
@@ -41,13 +41,20 @@ export default function makeDefaultEffects() {
   effects[HTTP] = function(props, getAction) {
     props.options = props.options || {}
     props.options.response = props.options.response || "json"
+    var errorAction = props.options.error || props.action
+    if (props.options.error) {
+      delete props.options.error
```

Comment:
Does having the `error` property in the options cause a problem? Should we be removing the `response` property also?

Reply:
I think we should, because we don't know how to work fetch. We also don't know how to fetch will envolve. If in a new version, "error" or "response" option is added to `fetch`, we will have side effects. This is cleaner to give to `fetch` only that `fetch` need.

Reply:
Actually, we should probably be making a copy of the object with properties removed instead of mutating the original. Something like `_.omit` from lodash.

Reply:
Yeah, I wouldn't pull in lodash just for that one function, but there's a few places I would use it if we get one.